docs_source/src/components/MenuIcon.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import classNames from "classnames";
+
+class MenuIcon extends Component {
+  render() {
+    const { isMenuHide, onClick } = this.props;
+
+    return (
+      <section id="menu-icon" onClick={onClick}>
+        <div
+          className={classNames("line", {

```

Comment:
hi @AhmedElbohoty, you can use another approach with string interpolation and with this avoid the module classNames, something like this `<div className={`line ${!isMenuHide && js-rotate-1}`}>`

I like this approach because the code doesn't depend on an external library